composer.json

```diff
@@ -17,7 +17,7 @@
     "require-dev": {
         "phpunit/phpunit": "4.2.*",
         "mockery/mockery": "0.9.*",
-        "illuminate/queue": "4.*"
```

Comment:
yes this is required, API dont have direct dependency but it is installed indirectly via dependency of laravel/framework(I am not sure exactly(